protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -584,6 +584,60 @@ definitions:
       - overwrite
       #- upsert_dedup # TODO chris: SCD Type 1 can be implemented later
       - append_dedup # SCD Type 1 & 2
+      - insert
+      - update
+      - upsert
+      - soft_delete
+  DiscoveredDestination:
+    type: object
+    additionalProperties: true
+    description: Return value for a discover command on a catalog. This should indicate which operations are supported for which objects.
+    required:
+      - stream_name
+      - operations
+    properties:
+      stream_name:
+        description: The name of the stream
+        type: string
```

Comment:
Are we intentionally only listing one stream here? Can a discovered destination not have multiple streams?

Reply:
You hare right and I'm being a big dummy. Let me update that and put everything under a `streams` field